fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -518,6 +531,16 @@ private Loan(final String accountNo, final Client client, final Group group, fin
         // rates added here
         this.rates = rates;
 
+        if (loanProduct.isRevolving()) {
+            if (revolvingPeriodStartDate != null) {
```

Comment:
why `!=` null?

Reply:
I leave the option to not define the revolving period, and it should mean that the credit can always revolve